au4/src/projectscene/CMakeLists.txt

```diff
@@ -49,6 +49,8 @@ set(MODULE_SRC
     ${CMAKE_CURRENT_LIST_DIR}/view/clipsview/waveview.h
     ${CMAKE_CURRENT_LIST_DIR}/view/clipsview/selectionviewcontroller.cpp
     ${CMAKE_CURRENT_LIST_DIR}/view/clipsview/selectionviewcontroller.h
+    ${CMAKE_CURRENT_LIST_DIR}/view/clipsview/gridlines.cpp
+    ${CMAKE_CURRENT_LIST_DIR}/view/clipsview/gridlines.h
```

Comment:
I think that logically gridlines belongs to timeline, it would be better to move it there.